MapboxNavigationUI/RouteMapViewController.swift

```diff
@@ -101,6 +99,20 @@ class RouteMapViewController: UIViewController, PulleyPrimaryContentControllerDe
         routePageViewController.notifyDidReRoute()
     }
     
+    @IBAction func toggleOverview(_ sender: Any) {
+        if isInOverviewMode {
+            recenterButton.isHidden = true
+            overviewButton.setTitle("Overview", for: .normal)
+            isInOverviewMode = !isInOverviewMode
```

Comment:
It would be clearer to either move this line out of the `if` statement (and remove the similar one from the `else` case) or explicitly say `false` here and `true` below.